### PR TITLE
Add an info popup and zoom buttons

### PR DESCRIPTION
**Pull request by IanButterworth**
See the new `i` icon here.

I wanted to add a colored legend too, but the flamegraph colors are dynamic, and showing colors in a gtk popup would take some work.

Screenshot from 2022-01-15 15-28-26

### CODE REVIEW

**Comment by IanButterworth** [comment]
I tried to add a test for this, but I can't find a way to programmatically close a `Gtk.info_dialog` given that the function doesn't return until the user clicks "OK". A timed async `destroy` might have worked, but felt a bit ugly for testing

**Comment by IanButterworth** [comment]
Also added:
- Zoom buttons
- Toolbar spacers
- Tooltips on the toolbar buttons

Screen Shot 2022-01-16 at 3 19 59 AM

File: src/ProfileView.jl [hunk 1]
```diff
@@ -353,6 +378,51 @@ function _save(selection, args...)
...
+
+    CTRL-clicking and dragging will zoom in on a specific region of the image. You can also control the zoom level with CTRL-scroll (or CTRL-swipe up/down).
+
+    CTRL-double-click to restore the full view.
+
+    You can pan the view by clicking and dragging, or by scrolling your mouse/trackpad (scroll=vertical, SHIFT-scroll=horizontal).
+
+    The toolbar at the top includes icons to load and save profile data. Clicking the save icon will prompt you for a filename; you should use extension *.jlprof for any file you save. Launching ProfileView.view(nothing) opens a blank window, which you can populate with saved data by clicking on the "open" icon.
+
+    After clicking on a bar, you can type warntype_last and see the result of code_warntype for the call represented by that bar.
```
Comment (timholy) [inline src/ProfileView.jl:416]:
```suggestion
    After clicking on a bar, you can type `warntype_last()` and see the result of `code_warntype` for the call represented by that bar.
```

File: src/ProfileView.jl [hunk 2]
```diff
@@ -353,6 +378,51 @@ function _save(selection, args...)
...
+    return nothing
+end
+
+@guarded function info_cb(::Ptr, ::Tuple)
+    # Note: Keep this updated with the readme
+    info = """
+    ProfileView.jl Interface Tips
+    ----------------------------------------------
+
+    `Ctrl-q` and `Ctrl-w `close the window. You can also use `ProfileView.closeall()` to close all windows opened by ProfileView.
```
Comment (timholy) [inline src/ProfileView.jl:402]:
```suggestion
    `Ctrl-q` and `Ctrl-w` close the window. You can also use `ProfileView.closeall()` to close all windows opened by ProfileView.
```

File: src/ProfileView.jl [hunk 3]
```diff
@@ -353,6 +378,51 @@ function _save(selection, args...)
...
+
+    Right-clicking on a bar calls the `edit()` function to open the line in an editor. (On a trackpad, use a 2-fingered tap.)
+
+    CTRL-clicking and dragging will zoom in on a specific region of the image. You can also control the zoom level with CTRL-scroll (or CTRL-swipe up/down).
+
+    CTRL-double-click to restore the full view.
+
+    You can pan the view by clicking and dragging, or by scrolling your mouse/trackpad (scroll=vertical, SHIFT-scroll=horizontal).
+
+    The toolbar at the top includes icons to load and save profile data. Clicking the save icon will prompt you for a filename; you should use extension *.jlprof for any file you save. Launching ProfileView.view(nothing) opens a blank window, which you can populate with saved data by clicking on the "open" icon.
```
Comment (timholy) [inline src/ProfileView.jl:414]:
```suggestion
    The toolbar at the top includes icons to load and save profile data. Clicking the save icon will prompt you for a filename; you should use extension *.jlprof for any file you save. Launching `ProfileView.view(nothing)` opens a blank window, which you can populate with saved data by clicking on the "open" icon.
```

File: src/ProfileView.jl [hunk 4]
```diff
@@ -353,6 +378,51 @@ function _save(selection, args...)
...
+
+    After clicking on a bar, you can type warntype_last and see the result of code_warntype for the call represented by that bar.
+
+    `ProfileView.view(windowname="method1")` allows you to name your window, which can help avoid confusion when opening several ProfileView windows simultaneously.
+
+    On Julia 1.8 `ProfileView.view(expand_tasks=true)` creates one tab per task. Expanding by thread is on by default and can be disabled with expand_threads=false.
```
Comment (timholy) [inline src/ProfileView.jl:420]:
```suggestion
    On Julia 1.8 `ProfileView.view(expand_tasks=true)` creates one tab per task. Expanding by thread is on by default and can be disabled with `expand_threads=false`.
```

**Review by timholy — COMMENTED** [review]
Amazing! Big step forward for usability.

Too bad about the difficulty of running the test. It would be nice if we could couple the `Close` button to an `Observable`, but that can be separate. We should merge this without the test since I think it's a strict improvement.

**Review by timholy — APPROVED** [review]
Merge when ready, and tag a new release.

**Pull request merged**